msi.c

```diff
@@ -1946,37 +1946,91 @@ static void dirtree_save(MSI_DIRENT *dirent, BIO *outdata, MSI_OUT *out)
     out->sectorNum += out->dirtreeSectorsCount;
 }
 
+static void fat_pad_last_sector(MSI_OUT* out, int padValue, char* buf) {
```

Comment:
The space before or after the asterisk is just a matter of aesthetics. When modifying an existing file, please use the style in that file.
```suggestion
static void fat_pad_last_sector(MSI_OUT *out, int padValue, char *buf)
{
```

Reply:
It would be great to add clang-format support or any other formatter, to not keep track of code style manually.